src/components/footer.jsx

```diff
@@ -48,56 +47,58 @@ function Footer() {
   const secondaryLinks = links.secondary;
 
   return (
-    <FooterUSWDS
-      size="medium"
-      // TODO: for some reason class name has to be manually added for returnToTop
-      returnToTop={(
-        <GridContainer>
-          <Grid className="usa-footer__return-to-top">
-            <Link to="#top">
-              {t('footer.returnToTop')}
-            </Link>
-          </Grid>
-          <div id="feedback-widget" className="footer__feedback-widget" lang={i18n.language} pageTitle={t('title')} endpoint={process.env.GATSBY_ENDPOINT} />
-          <ScriptTag src="https://d2ttz3as5y3dj0.cloudfront.net/feedback-module.min.js" type="text/javascript" />
-        </GridContainer>
-      )}
-      primary={(
-        <FooterNav
-          size="medium"
-          links={primaryLinkItems}
-        />
-      )}
-      secondary={(
-        <Grid row gap>
-          <Logo
-            size="medium"
-            image={(
-              <img
-                className="footer__logo"
-                src={logoFooter}
-                alt={t('agency.longformName')}
-              />
-            )}
+    <div>
+      <div className="footer__return-to-top-feedback">
+        <div className="usa-footer__return-to-top">
+          <Link to="#top">{t('footer.returnToTop')}</Link>
+        </div>
+        <div className="footer__feedback">
+          <div
+            id="feedback-widget"
+            lang={i18n.language}
+            pageTitle={t('title')}
+            endpoint={process.env.GATSBY_ENDPOINT}
           />
-          <Grid
-            className="usa-footer__contact-links footer"
-            mobileLg={{ col: 6 }}
-          >
-            <Grid className="footer__copyright-trademark">
-              <p>{t('footer.copyright')}</p>
-              <p>{t('footer.trademark')}</p>
-            </Grid>
-            <Grid className="footer__terms">
-              <Link to={secondaryLinks.terms}>{t('footer.terms')}</Link>
-            </Grid>
-            <Grid className="footer__privacy">
-              <Link to={secondaryLinks.privacy}>{t('footer.privacy')}</Link>
+          <ScriptTag
+            src="https://d2ttz3as5y3dj0.cloudfront.net/feedback-module.min.js"
+            type="text/javascript"
+          />
+        </div>
+      </div>
+      <FooterUSWDS
+        size="medium"
+        primary={<FooterNav size="medium" links={primaryLinkItems} />}
+        secondary={(
+          <Grid row gap>
+            <Logo
+              size="medium"
+              image={(
+                <img
+                  className="footer__logo"
+                  src={logoFooter}
+                  alt={t('agency.longformName')}
+                />
+              )}
+            />
+            <Grid
+              className="usa-footer__contact-links footer__info"
```

Comment:
Is `usa-footer__contact-links` a relevant name?

Reply:
It applies some styles from USWDS. We could remove it and manually add styles to footer__info

Reply:
Yes, I think that would be better.

Reply:
Let's discuss use of uswds classes and custom classes in code review. I'll address in a future PR once we do